exporters/jaeger/README.md

```diff
@@ -31,28 +31,31 @@ Then, configure the SDK to use the Jaeger exporter as a span processor, and use
 require 'opentelemetry/sdk'
 require 'opentelemetry/exporters/jaeger'
 
-# Create a concrete tracer factory
-factory = OpenTelemetry::SDK::Trace::TracerFactory.new
-
-# Configure the tracer factory
-factory.add_span_processor(
-  OpenTelemetry::SDK::Trace::Export::SimpleSpanProcessor.new(
-    OpenTelemetry::Exporters::Jaeger::Exporter.new(
-      service_name: 'my-service', host: 'localhost', port: 14268
-    )
+# initialize a SimpleSpanProcessor with Jaeger export
+processor = OpenTelemetry::SDK::Trace::Export::SimpleSpanProcessor.new(
+  OpenTelemetry::Exporters::Jaeger::Exporter.new(
+    service_name: 'my-service', host: 'localhost', port: 6831
```

Comment:
I'd pull all this down into the `...SDK.configure` block. `processor` isn't used anywhere else.